test/09_contracts.ts

```diff
@@ -49,8 +57,14 @@ describe('contracts', () => {
 		await removeContractDirectory();
 		await clearDeviceTypeResource();
 
-		mockRepo('balena-io', 'contracts', 'base-contracts', true);
-		await synchronizeContracts([{ owner: 'balena-io', name: 'contracts' }]);
+		mockRepo(
+			contractRepository.owner,
+			contractRepository.name,
+			contractRepository.branch,
+			'base-contracts',
+			true,
+		);
```

Comment:
NIT: Maybe it now makes sense to convert the arguments of this into an object, to make obvious what each arg is when invoking it and not have to manually spread all properties of the RepositoryInfo.
Eg:
```suggestion
		mockRepo({
			...contractRepository,
			filename: 'base-contracts',
			persist: true,
		});
```
Definitely doesn't have to be in this PR though.

Reply:
Improved this method's parameters in latest push.